Mlem/API/Models/Community/APICommunityModeratorView.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  APICommunityModeratorView.swift
+//  Mlem
+//
+//  Created by Nicholas Lawson on 09/06/2023.
+//
+
+import Foundation
+
+struct APICommunityModeratorView: Decodable {
+    let community: APICommunity
+    let moderator: APIPerson
+}
+
+extension APICommunityModeratorView: Identifiable {
+    var id: Int { moderator.id }
```

Comment:
It's a tuple of community and moderator, so this'll incorrectly equate when you're comparing with a different community

Reply:
ah! good catch, thanks, I'll update

Reply:
fixed in 5cf9c1ef79e45e6b56bfcecc79afeb9edb8ab916